### DISCUSSION TITLE
How to completely remove the app icon?

### DISCUSSION THREAD

**Discussion by Zingzy** [TomSchimansky/CustomTkinter#1144] [Q&A]
Hi, I am using Custom tkinter version 5.0.2 which has the default tkinter icon. I want to completely remove the icon and also want the title to completely fill the icons place. I tried many things even I placed a transparent icon but it didn't full suffice my needs. Please help me out.

**Comment by Anirudh-12** [comment]
U can Remove the title bar totally and make a custom title bar
Code to remove the title bar
```
root = CTk()
root.overrideredirect(True)
root.mainloop()
```

**Reply by Akascape** [reply]
I have made one ready to use: https://github.com/TomSchimansky/CustomTkinter/discussions/736

**Comment by Anirudh-12** [comment]
this will remove app icon totally 
```
root = CTk()
root.wm_attributes('-toolwindow', 'True')
root.mainloop()
```
